lua/entities/gmod_wire_expression2/core/custom/cfc_entity.lua

```diff
@@ -52,4 +52,43 @@ e2function void entity:ejectPodTo(vector pos)
 
     driver:ExitVehicle()
     driver:SetPos(clampedPos)
-end
\ No newline at end of file
+end
+
+e2function array entities()
```

Comment:
pretty sure this function could also cause a lot of lag.

Reply:
tested it, it doesnt, atleast on singleplayer.

Reply:
From my testing, with the added quota line thingy, it's performance impact doing `runOnTick(1)
while(perf()) {
entities()
}
` is the same as doing `runOnTick(1)
while(perf()) {
    findInSphere(entity():pos(),5000000)
    findToArray()
}
`

Reply:
Both forms max out the ops quota of the e2 in that form, as well as the entities() function causing less cpu time, on average.

Reply:
Might as well increase the costs of both functions, just in case.